src/components/Header.js

```diff
@@ -3,7 +3,6 @@ import styled, { css } from "styled-components";
 import { Link } from "react-router-dom";
 import LogoIcon from "../icons/LogoIcon";
 import {
```

Comment:
I am unsure if it's a good convention to import from Storybook, please check that